### DISCUSSION TITLE
resize via mouse?

### DISCUSSION THREAD

**Discussion by bobblkabb** [leftwm/leftwm#1180] [Q&A] [answered]
Is there a way to resize windows by Alt + right mouse click?

**Comment by Souliboi** [comment, marked as answer]
by default you should be able to resize using Mod+Right mouse key, Mod being the Super(Windows) key by default i believe
though it should be noted that the window immediately gets classified as floating when you resize it that way

**Reply by bobblkabb** [reply]
No, it doesn't. 
Got btw:
mousekey: "Mod1"
..

**Comment by VuiMuich** [comment]
Iirc there is an option to set the modifier key for mouse combos. Could be `mouse_mod`, but not sure from the top of my head. It should be mentioned in the wiki though, just can't look up the link for you right now.

**Reply by bobblkabb** [reply]
Couldn't find it in the wiki.

**Reply by VuiMuich** [reply]
:thinking: I thought of this [`Mousekey`](https://github.com/leftwm/leftwm/wiki/Config#mousekey), but when it does not work, there might also be some misconception by the wiki itself or an actual bug.
Have you tried `mousekey: "Alt"` instead of `Mod1`?

**Reply by Eskaan** [reply]
@VuiMuich see the [other thread](https://github.com/leftwm/leftwm/discussions/1180#discussioncomment-7596723), I guess we can mark this as answered.

**Comment by bobblkabb** [comment]
> by default you should be able to resize using Mod+Right mouse key, Mod being the Super(Windows) key by default i believe though it should be noted that the window immediately gets classified as floating when you resize it that way
So there's no way to resize without making it float?!

**Reply by Eskaan** [reply]
No, if you resize a window it always starts to float.

**Reply by bobblkabb** [reply]
Think I turned it off by setting:
`disable_tile_drag: true`

That explains it.
